### ISSUE TITLE
Incorrectly reporting "Unclosed block" on SCSS mixin

### ISSUE THREAD

**Issue by ConduciveMammal** [shinnn/vscode-stylelint#141]
I have a SCSS mixin for input placeholders and this extension reports that it's incorrect, but no other compiler/linter reports an issue?
The mixin is
```
@mixin placeholder {
  &::-webkit-input-placeholder {@content}
  &:-moz-placeholder           {@content}
  &::-moz-placeholder          {@content}
  &:-ms-input-placeholder      {@content}
}
```

It's reporting: `Unclosed block (CssSyntaxError)` and seems to be stemming from the part: `  &:-moz-placeholder           {@content}`.

**Comment by shinnn** [comment]
This is not an issue directly related to this repository.
